exec/java-exec/src/main/java/org/apache/drill/exec/physical/impl/join/HashJoinMemoryCalculatorImpl.java

```diff
@@ -234,6 +235,7 @@ public HashJoinState getState() {
     private int recordsPerPartitionBatchBuild;
     private int recordsPerPartitionBatchProbe;
     private int outputBatchNumRecords;
+    private int outputBatchSize;
     private Map<String, Long> buildValueSizes;
     private Map<String, Long> probeValueSizes;
     private Map<String, Long> keySizes;
```

Comment:
@ppadma I think outputBatchNumRecords, buildValueSizes, probeValueSizes, and keySizes are unused now since we are directly passing in the outputBatchSize. This is great since the calculator has been simplified. Could you also delete these unused parameters?